app/assets/javascripts/video-player.js.es6

```diff
@@ -67,9 +67,18 @@
           const originalOnPlayerStateChange = this.onPlayerStateChange;
 
           this.onPlayerStateChange = function(stateId) {
+            let stateValue = 'UNKNOWN';
+
+            $.each(YT.PlayerState, function(key, value) {
+              if (value === stateId) {
+                stateValue = key;
+                return;
```

Comment:
Tiny thing but I think we need to actually `return false;` out of this: http://stackoverflow.com/questions/1799284/how-to-break-exit-from-a-each-function-in-jquery